purchases/src/main/kotlin/com/revenuecat/purchases/paywalls/components/ButtonComponent.kt

```diff
@@ -25,6 +25,7 @@ import kotlinx.serialization.encoding.Encoder
 class ButtonComponent(
     @get:JvmSynthetic val action: Action,
     @get:JvmSynthetic val stack: StackComponent,
+    @get:JvmSynthetic val transition: PaywallTransition? = null,
```

Comment:
Nothing to do here but one thing I was seeing is that we won't allow to override the animation for different screen sizes or other parameters. Having said that, we also don't have overrides for button actions, so I think this is more than likely ok, and I don't think there would be many use cases for us to override a button action or transition depending on screen sizes or other parameters, at least for now.

Reply:
And if the user DID want to do that, they could just conditionally show the button they want on a phone and another on a tablet with different actions or transitions.